.goreleaser.yml

```diff
@@ -46,5 +46,4 @@ dockers:
       - --label=org.label-schema.description='ct - The chart testing tool'
       - --label=org.label-schema.vendor=Helm
     extra_files:
-      - etc/chart_schema.yaml
-      - etc/lintconf.yaml
+      - etc
```

Comment:
I'm getting the following error when running `build.sh` locally:

```
COPY failed: stat /var/lib/docker/tmp/docker-builder595575462/etc/chart_schema.yaml: no such file or directory
```

This fixes that, but I'm not sure why. 😦

Reply:
Hm, I can run it successfully and so can CircleCI. Not sure why it fails for you.

Reply:
on my mac it failed as well

Reply:
I'm also on a mac, goreleaser installed via homebrew. Looks like my version is a little old though:

```
0.93.2, commit ce267acabf3205e889ead098a589d905bef32d01, built at 2018-11-12T12:08:25Z
```

I'll try installing a more recent version.

Reply:
Yeah, no longer a problem with binary installed from https://github.com/goreleaser/goreleaser/releases/tag/v0.101.0.

Should we keep this fix for homebrew users anyway?